app/javascript/crayons/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,35 @@
+import { h, cloneElement } from 'preact';
+import classNames from 'classnames/bind';
+
+export const Tabs = (props) => {
+  const {
+    children,
+    stacked,
+    fitted,
+    className,
+    elements = 'buttons',
+    ...otherProps
+  } = props;
```

Comment:
```suggestion
export const Tabs = ({
    children,
    stacked,
    fitted,
    className,
    elements = 'buttons',
    ...otherProps
  }) => {
```

Just a suggestion again